tests/smartcam/modules/test_matter.py

```diff
@@ -0,0 +1,20 @@
+from kasa import Module
+from kasa.smart import SmartDevice
+
+from ...device_fixtures import parametrize
+
+matter = parametrize(
+    "has matter", component_filter="matter", protocol_filter={"SMARTCAM"}
```

Comment:
Even when this is not (yet?) a standard interface, perhaps it makes no sense to duplicate the test code but just filter on both protocols?

Reply:
That's done